SrMoffat/Book-A-Meal
# [Story#157110156] --> Merge ch-test-meal-creation-157059078 to develop (Tests for endpoints)

**What does the PR do?**
On-board the unit tests for the API endpoints as per product requirements 
** Task Completed**
- [ Done] Test for /api/v1/auth/signup endpoint 
- [Done ] Test for api/v1/auth/login endpoint 
- [Done ] Test for POST api/v1/meals endpoint
- [Done ] Test for GET api/v1/meals endpoint
- [Done ] Test for PUT api/v1/meals endpoint
- [Done ] Test for DELETE api/v1/meals endpoint
- [ Done] Test for POST api/v1/menu endpoint 
- [ Done] Test for GET api/v1/menu endpoint
- [ Done] Test for POST api/v1/orders endpoint 
- [Done ] Test for PUT api/v1/orders endpoint
- [Done ] Test for GET api/v1/orders endpoint 
**Manual Testing**
Download or clone the repository's ch-test branch, navigate to the `/Book-A-Meal` project folder, within it there is a file under the name `test_book_a_meal.py` , run it using the command python `test_book_a_meal.py.` Expected results (13 tests to run --> all of them fail)
**Background**
Give the blocker I had yesterday (communicated to team lead), I have included all the tests in the same branch under the name `ch-test-meal-creation-157059078` . Even though the branch name suggests that the test is only for meal creation, be aware that the tests for all endpoints have been included. 
**Pivotal Tracker Stories**
#157059178
#157059078
#157078550
#157078511
#157059224
#157058934
#157059005
#157059801
#157059536
#157059459
#157059285
#157059377
